src/commands/install-packages.yml

```diff
@@ -0,0 +1,77 @@
+description: >
+  Install your composer packages with automated caching and best practices applied.
+
+parameters:
+  app-dir:
+    type: string
+    default: ~/project
+    description: Path to the directory containing your composer.json file. Not needed if composer.json lives in the root.
+  install-command:
+    type: string
+    default: composer install --no-interaction --prefer-dist
```

Comment:
Maybe we change this to `install-flags`, making `composer install << parameters.install-flags>>` the command for the run step.

Reply:
done in https://github.com/CircleCI-Public/php-orb/pull/25/commits/03cd910d2478f5bb5d6e33174551ce1d030a326d